### DISCUSSION TITLE
error emqx_auth_mysql

### DISCUSSION THREAD

**Discussion by mariamendez01** [emqx/emqx#6914] [Q&A] [answered]
Hi, 
I need a lot help

I'm trying to start my plugin **emqx_auth_mysql** 

![mysql_error](https://user-images.githubusercontent.com/85718544/152223492-554f3118-40fd-4b0a-bd1e-ce9f2c9989d4.png)

but at the moment to push START, just appear this label that says: parse-config_file_failed.

I was trying to put in the folder ogf plugins and even start from there, but also an error.

![mysql_error_1](https://user-images.githubusercontent.com/85718544/152224212-7867e459-c8f0-4440-b17c-36e3f5418315.png)

**Comment by mariamendez01** [comment]
![mysql_error_2](https://user-images.githubusercontent.com/85718544/152228032-8691ae26-67c6-4d5e-adbc-85cf41ff0e42.png)

even now when I want stop/start EMQX I see this: 
the file is damaged?

**Comment by zmstone** [comment]
@mariamendez01 
which EMQ X version ?
could you paste `data/loaded_plugins` file content ?

**Reply by mariamendez01** [reply]
4.2.13 version

and yes, maybe I l could paste again the file. This will be correct or will create an error?

**Reply by zmstone** [reply]
i mean paste the content here is so we can help to inspect it

**Reply by mariamendez01** [reply]
![mysql_error_3](https://user-images.githubusercontent.com/85718544/152243723-74e26e20-42bc-4185-986f-92cb78597d79.png)
 of course, here is

**Reply by zmstone** [reply]
The last line is missing a dot at the end.

**Reply by mariamendez01** [reply]
oh, thats rigth, but even if I put the dot at the end of the line, my server behavior is the same as the first screenshot of parse_config_file_failed

**Reply by zmstone** [reply]
Plugin config is not as expected.
Please provide the full log lines (but not screenshot)
And the enqx_auth_mysql.conf file content

**Reply by mariamendez01** [reply]
This is everything in this file:
/emqx/data/nano loaded_plugins


emqx_management.
emqx_recon.
emqx_retainer.
emqx_dashboard.
emqx_telemetry.
emqx_rule_engine.
emqx_auth_mysql.

**Reply by mariamendez01** [reply]
and this is all in the file emqx_auth_mysql.conf


```


##--------------------------------------------------------------------
## MySQL Auth/ACL Plugin
##--------------------------------------------------------------------

## MySQL server address.
##
## Value: Port | IP:Port
##
## Examples: 3306, 127.0.0.1:3306, localhost:3306
auth.mysql.server = 127.0.0.1:3306

## MySQL pool size.
##
## Value: Number
auth.mysql.pool = 8

## MySQL username.
##
## Value: String
auth.mysql.username = emqx

## MySQL password.
auth_mysql.password = public

## Value: String
## auth.mysql.password =

## MySQL database.
##
## Value: String
auth.mysql.database = mqtt

## MySQL query timeout
##
## Value: Duration
auth.mysql.query_timeout = 5s

## Variables: %u = username, %c = clientid

```

**Reply by mariamendez01** [reply]
I saw in this video that he stabich the configurations in this fields and is very similar to the configurations in the file **emqx_auth_mysql.conf**
 
so.. is possible if I make the same changes that I was expecting to do in the dashboard, but now just make the changes in the console?

But, then what about the error in the dashboard about the parse_config_file_failed? :c


https://www.youtube.com/watch?v=kjd18IoFujU

**Reply by mariamendez01** [reply]
and I want put this status in **active=TRUE** 


![mysql_error_3](https://user-images.githubusercontent.com/85718544/152252788-ea1d1cb6-3eb8-4f41-827f-8a1129b46a8f.png)

**Comment by zmstone** [comment, marked as answer]
auth_mysql.password = public
should be
auth.mysql.password = public
Instead

Btw. We have stopped maintaining 4.2 community edition. Please consider 4.3
